reflex/event.py

```diff
@@ -462,7 +511,9 @@ def call_event_fn(fn: Callable, arg: Var) -> list[EventSpec]:
     return events
 
 
-def get_handler_args(event_spec: EventSpec, arg: Var) -> tuple[tuple[Var, Var], ...]:
+def get_handler_args(
+    event_spec: EventSpec, arg: Var | ArgsSpec
```

Comment:
does this function even use `arg`?

Reply:
It doesn't, I just kept the signature since `arg` was passed beforehand, but we could remove it I think.